compositor_render/src/state.rs

```diff
@@ -192,34 +211,34 @@ impl InnerRenderer {
         self.scene
             .register_render_event(inputs.pts, input_resolutions);
 
-        populate_inputs(ctx, &mut self.render_graph, &mut inputs).unwrap();
-        run_transforms(ctx, &mut self.render_graph, inputs.pts).unwrap();
-        let frames = read_outputs(ctx, &mut self.render_graph, inputs.pts).unwrap();
+        let pts = inputs.pts;
+        populate_inputs(ctx, &mut self.render_graph, inputs);
+        run_transforms(ctx, &mut self.render_graph, pts);
+        let frames = read_outputs(ctx, &mut self.render_graph, pts);
 
         scope.pop(&ctx.wgpu_ctx.device)?;
 
-        Ok(FrameSet {
-            frames,
-            pts: inputs.pts,
-        })
+        Ok(FrameSet { frames, pts })
     }
 
     pub fn update_scene(
         &mut self,
-        scenes: Vec<scene::OutputScene>,
+        outputs: Vec<scene::OutputScene>,
     ) -> Result<(), UpdateSceneError> {
-        let output_nodes =
-            self.scene
-                .update_scene(scenes, &self.renderers, &self.text_renderer_ctx)?;
-        self.render_graph.update(
-            &RenderCtx {
-                wgpu_ctx: &self.wgpu_ctx,
-                text_renderer_ctx: &self.text_renderer_ctx,
-                renderers: &self.renderers,
-                stream_fallback_timeout: self.stream_fallback_timeout,
-            },
-            output_nodes,
-        )?;
+        for output_scene in outputs {
```

Comment:
tmp, in a follow up PR the update_scene will be modified to only update one scene